Survival-DEV/blog-backend
# Blog 28: fix CSRF & CORS

### :dart:  What is been done in this PR:
- [x] Fix the `interanl server error` while `POST/PUT` ~ 
related to `csrf`
- [x] use Redis as storage for throttler 

**Relates to #28**

----
### :compass:   How to test it
Steps to test my PR:
1. `npm run ci`
2. make sure to setup your `env` file to have these params:
> DATABASE_URL=<your db url#>
> NODE_ENV=DEBUG  //for now
> logDB=["query", "error"]
> PORT=<your port#>
> TRUSTED_IP=["127.0.0.1"]
> WHITELIST=['http://localhost']
>ALLOWED_METHODS='GET,HEAD,PUT,PATCH,POST,DELETE,OPTIONS'
4. `docker-compose up`   --to run redis 
3. `npm run start:dev`   --in another terminal
